src/scss/global.scss

```diff
@@ -1,12 +1,7 @@
-@import "constants.scss";
-
-:root {
-  --primary-color: #f6e652;
-}
-
 @import "node_modules/bootswatch/dist/darkly/variables";
 @import "node_modules/bootstrap/scss/bootstrap";
 @import "node_modules/bootswatch/dist/darkly/bootswatch";
+@import "./colors.scss";
 @import "./constants.scss";
```

Comment:
All files just including `constants.scss` will not include `colors.scss`. This depends on global styles thus will show warning/error underlines in IDEs because it isn't directly included.

Suggest include `colors.scss` as first line in `constants.scss`

Reply:
Well, `colors.scss` contains some 

> :root {...} 

enhancements with color vars. It's included in `global.scss` file and that one is imported once in `src/index.tsx`. 

When I'm trying to add `colors.scss` as the first line in `constants.scss` => it's imported dozen of times
<img width="1440" alt="Screenshot 2021-10-04 at 13 16 48" src="https://user-images.githubusercontent.com/26366773/135834350-384e8579-1f80-4135-a71f-9342229fc952.png">

Reply:
This can only mean there could be a split between the single-include and multiple-include parts of the `colors.scss`.

Reply:
All good I guess, I now notice the split was already done in https://github.com/sparkletown/sparkle/pull/2343/files#diff-e81ecc2f161d11a623a02514686e44a4a4b3fb7a08a5c3b3fecab9895652ab95R1-R9

Reply:
Good work!